src/native/libs/System.Native/pal_runtimeinformation.h

```diff
@@ -23,6 +23,7 @@ enum
     ARCH_ARM64,
     ARCH_WASM,
     ARCH_S390X,
+    ARCH_POWERPC,
```

Comment:
> The order has to be same as System.Runtime.InteropServices.Architecture enum

This needs to be added as last item here.

Reply:
updated